internal/nursery/swap.go

```diff
@@ -293,7 +317,9 @@ func (nursery *Nursery) handleSwapStatus(swap *database.Swap, status boltz.SwapS
 		return
 	}
 
-	if parsedStatus.IsCompletedStatus() {
+	// dont wait for boltz to claim the swap in case of batchOnly
+	// in which case it will stay at transaction.claimed for longer
```

Comment:
```suggestion
	// Don't wait for boltz to claim the swap in case of batchOnly
	// in which case it will stay at transaction.claim.pending for longer
```